Blockzilla/Utilities/OnboardingTelemetryHelper.swift

```diff
@@ -0,0 +1,100 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Glean
+
+enum CardViewType: String {
+    case welcomeView = "welcome"
+    case defaultBrowserView = "default-browser"
+    case widgetTutorial = "widget-tutorial"
+}
+
+class OnboardingTelemetryHelper {
+    public enum Event {
+        case getStartedAppeared
+        case getStartedCloseTapped
+        case getStartedButtonTapped
+        case defaultBrowserCloseTapped
+        case defaultBrowserSettingsTapped
+        case defaultBrowserSkip
+        case defaultBrowserAppeared
+        case widgetCardAppeared
+        case widgetPrimaryButtonTapped
+        case widgetCloseTapped
+    }
+
+    func handle(event: Event) {
+        switch event {
+        case .getStartedAppeared:
+            onboardingGetStartedScreenDisplayed()
+        case .getStartedCloseTapped:
+            onboardingGetStartedScreenDismiss()
+        case .getStartedButtonTapped:
+            onboardingGetStartedScreenButtonTapped()
+        case .defaultBrowserCloseTapped:
+            onboardingDefaultBrowserScreenDismiss()
+        case .defaultBrowserSettingsTapped:
+            onboardingDefaultBrowserScreenButtonTapped()
+        case .defaultBrowserSkip:
+            onboardingDefaultBrowserScreenSkipTapped()
+        case .defaultBrowserAppeared:
+            onboardingDefaultBrowserScreenDisplayed()
+        case .widgetCardAppeared:
+            onboardingWidgetScreenDisplayed()
+        case .widgetPrimaryButtonTapped:
+            onboardingWidgetPrimaryActionClicked()
+        case .widgetCloseTapped:
+            onboardingWidgetScreenDismiss()
+        }
+    }
+
+    func onboardingGetStartedScreenButtonTapped() {
```

Comment:
Since most of the code is one liner I think we can get rid of intermediate methods.